### ISSUE TITLE
Styling a RectButton

### ISSUE THREAD

**Issue by mydesweb** [kmagiera/react-native-gesture-handler#669]
I just observed that setting borderStyle in style prop of a RectButton has no effect on iOS at least, I think not all styles are supported yet.

**Comment by dan10gc** [comment]
Which version of `react-native-gesture-handler` are you using?

**Comment by mydesweb** [comment]
"react-native-gesture-handler": "^1.3.0",

**Comment by rszalski** [comment]
@mydesweb take a look at https://github.com/kmagiera/react-native-gesture-handler/issues/675#issuecomment-512742226 I think it’s the same issue underneath (overflow). 

Probably borders are rendered outside the element and they are clipped on iOS, same as shadows etc.

**Comment by mydesweb** [comment]
@rszalski you are right, thanks for explanations.. borders are rendered outside and overflow is set to hidden by default..
